toolz/itertoolz.py

```diff
@@ -26,7 +26,7 @@ def remove(predicate, seq):
     return filterfalse(predicate, seq)
 
 
-def accumulate(binop, seq):
+def accumulate(binop, seq, start=no_default):
```

Comment:
`reduceby` calls this `init`, should we stick with `initial` here and change that in a seperate pr or leave as is?